qiskit-superstaq/qiskit_superstaq/superstaq_backend.py

```diff
@@ -49,22 +56,27 @@ def validate_target(target: str) -> None:  # pylint: disable=missing-function-do
 
     prefix, _, device_type = match.groups()
 
-    # Check valid prefix
     if prefix not in vendor_prefixes:
         raise ValueError(
             f"{target} does not have a valid target prefix. "
             f"Valid target prefixes are: {vendor_prefixes}."
         )
 
-    # Check for valid device type
     if device_type not in target_device_types:
         raise ValueError(
             f"{target} does not have a valid target device type. "
             f"Valid target device types are: {target_device_types}."
         )
 
 
-class SuperstaQBackend(qiskit.providers.BackendV1):  # pylint: disable=missing-class-docstring
+class SuperstaQBackend(qiskit.providers.BackendV1):
+    """This class represents a Superstaq backend.
+
+    Args:
```

Comment:
Curious about the standard we follow: do we include the docstring for the constructor args in the class's `__init__` or in the class docstring?

Reply:
I went back and forth about this too! After a bit of googling and reviewing other parts of Superstaq, I went with putting everything right under the class definition. 

It might be a minor detail, but for consistency, this could be something worth discussing during dev meeting?

Reply:
Ah definitely missed this conversation while OOO! But this sounds good! I'll move the constructor arguments within ``__init__``. Will also make a note that this styling should go in the dev guide.